### PR TITLE
[MIG]Product brand

### PR DESCRIPTION
**Pull request by flotho**

### CODE REVIEW

File: product_brand/product_brand_view.xml [hunk 1]
```diff
@@ -129,18 +129,19 @@
...
             <field name="name">product.search.form</field>
             <field name="model">product.product</field>
             <field name="inherit_id" ref="product.product_search_form_view"/>
             <field name="arch" type="xml">
-                <field name="categ_id" position="after">
+                <field name="name" position="after">
                     <field name="product_brand_id"/>
                 </field>
-                <group string='Group by...' position="inside">
+<!--                <group string='Group by...' position="inside">
```
Comment (pedrobaeza) [inline product_brand/product_brand_view.xml]: Use `<filter name="template_id" posititon="after">` instead

File: product_brand/product_brand_view.xml [hunk 2]
```diff
@@ -152,10 +153,10 @@
                 <field name="name" position="after">
                     <field name="product_brand_id"/>
                 </field>
-                <group string='Group by...' position="inside">
+<!--                <group string='Group by...' position="inside">
```
Comment (pedrobaeza) [inline product_brand/product_brand_view.xml]: Use `<group posititon="inside">` instead. It will find the only group in the search

**Comment by pedrobaeza** [comment]
Bump version number

**Comment by LeartS** [comment]
Why does runbot report as successful when the build stopped with an (unrelated) error?

**Comment by lasley** [comment]
What is the status of this PR?

**Comment by rvalyi** [comment]
Hello repo PSC, I think this PR can be closed. Indeed product_brand is available in 9.0
https://github.com/OCA/product-attribute/tree/9.0/product_brand
cc @pedrobaeza

**Pull request closed without merging**